next/platform/macos/macos.cmake

```diff
@@ -165,24 +165,34 @@ add_executable(
     ${MBGL_ROOT}/platform/default/src/mbgl/test/main.cpp
 )
 
+target_include_directories(
+    mbgl-test-runner
+    PUBLIC ${MBGL_ROOT}/include ${MBGL_ROOT}/test/include
+)
+
 target_compile_definitions(
     mbgl-test-runner
     PRIVATE WORK_DIRECTORY=${MBGL_ROOT}
 )
 
 target_link_libraries(
     mbgl-test-runner
-    PRIVATE mbgl-compiler-options mbgl-test
+    PRIVATE mbgl-compiler-options -Wl,-force_load mbgl-test
 )
 
 add_executable(
     mbgl-benchmark-runner
     ${MBGL_ROOT}/platform/default/src/mbgl/benchmark/main.cpp
 )
 
+target_include_directories(
+    mbgl-benchmark-runner
+    PUBLIC ${MBGL_ROOT}/benchmark/include
+)
+
```

Comment:
Did we forget to add it in previous PRs?

Reply:
Yes